app/controllers/api/offenders_controller.rb

```diff
@@ -1,18 +1,28 @@
 # frozen_string_literal: true
 
-module Api
-  class OffendersController < Api::ApiController
-    respond_to :json
+class Api::OffendersController < Api::ApiController
+  respond_to :json
 
-    def show
-      @offender = OffenderService.get_offender(offender_number)
-      render_404('Not found') if @offender.nil?
+  def show
+    @offender = OffenderService.get_offender(offender_number)
+    if @offender.nil?
+      render_404('Not found')
+    else
+      render json: offender_as_json(@offender)
     end
+  end
 
-  private
+private
 
-    def offender_number
-      params.require(:nomis_offender_id)
-    end
+  def offender_number
+    params.require(:nomis_offender_id)
+  end
+
+  def offender_as_json(offender)
```

Comment:
Shouldn't these keys be camel case if it's JSON?

Reply:
That's the 'old' API that I don't want to touch until I've understood who's using it. But yes, ideally.